user_modules/script_runner.lua

```diff
@@ -90,7 +153,25 @@ function runner.Title(titleText)
 end
 
 function runner.Step(testStepName, testStepImplFunction, paramsTable)
-  extendedAddTestStep(testStepName, testStepImplFunction, paramsTable)
+  local maxLength = 101
+  local filler = "="
+  if isInitialStep then
+    printTestInformation()
+    if not isTestApplicable(runner.testSettings.restrictions.sdlBuildOptions) then
+      local message = "Test is incompatible with current build configuration of SDL:\n" ..
+          commonFunctions:convertTableToString(Test.sdlBuildOptions, 1) ..
+          "\n\nTest is compatible with SDL built with next build options:\n" ..
```

Comment:
@aderiabin, Please correct "Test is compatible with SDL built with ..." --> "Test is compatible with SDL with ..."

Reply:
@dboltovskyi Rephrased in e1265d7